backend/cmm_helpers.ml

```diff
@@ -1161,19 +1161,19 @@ let unboxed_mutable_float_array_ref arr ofs dbg =
 let unboxed_immutable_float_array_ref arr ofs dbg =
   Cop (mk_load_immut Double, [array_indexing log2_size_float arr ofs dbg], dbg)
 
-let unboxed_mutable_float32_unboxed_product_array_ref arr index dbg =
+let unboxed_mutable_float32_unboxed_product_array_ref arr ~element_num dbg =
```

Comment:
as discussed on slack: maybe `array_index` might be a slightly better label

Reply:
Renamed